vcs/git/repo.go

```diff
@@ -104,13 +104,13 @@ func (r *Repository) ResolveTag(name string) (vcs.CommitID, error) {
 	return "", vcs.ErrTagNotFound
 }
 
-func (r *Repository) Branches() ([]*vcs.Branch, error) {
+func (r *Repository) Branches(_ vcs.BranchesOptions) (branches []*vcs.Branch, total uint, err error) {
```

Comment:
Thanks. I am aware it's possible to avoid it, but I purposefully included the blank identifier as an indicator that the parameter is completely unused.